hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -600,7 +602,8 @@ private void initializeSystemManagers(OzoneConfiguration conf,
           eventQueue,
           scmContext,
           serviceManager,
-          scmNodeManager);
+          scmNodeManager,
+          new MonotonicClock(ZoneId.of("UTC")));
```

Comment:
Is there any reason not to use `ZoneOffset.UTC`?

Reply:
No reason, except I didn't know about it :-) I will change it.